### DISCUSSION TITLE
[newbies] ways of pulling data from LRGS servers.

### DISCUSSION THREAD

**Discussion by SebastienCharland** [opendcs/.github#4] [General]
Hi,

We're planning on rewriting our Windows LRGS data pull service based on DDS protocol 14, to something that will run on RedHat.
Hopefully in bash/curl, bash/sftp-rsync with sshkey, or python/libcurl.

Anyone can point me to documents on what is available / possible?

**Comment by MikeNeilson** [comment]
https://github.com/opendcs/opendcs/discussions/790
